Shell/Statistics.hpp

```diff
@@ -260,6 +266,8 @@ class Statistics {
   unsigned finalActiveClauses;
   /** extensionality clauses at the end of the saturation algorithm run */
   unsigned finalExtensionalityClauses;
+  /** max number of literals of a generated clause  */
+  unsigned biggestGeneratedClause;
```

Comment:
`biggestGeneratedClause` to be "dead".

Should we make it work or rather properly get rid off?

Reply:
removed it!